website/database.py

```diff
@@ -8,7 +8,7 @@
 TOKENS = dynamo.Table(storage, 'tokens', 'id')
 PROGRAMS = dynamo.Table(storage, 'programs', 'id', indexed_fields=['username'])
 CLASSES = dynamo.Table(storage, 'classes', 'id', indexed_fields=['teacher', 'link'])
-PREFERENCES = dynamo.Table(storage, 'preferences', 'id')
+CUSTOMIZATIONS = dynamo.Table(storage, 'class_customizations', partition_key='id', sort_key='level')
```

Comment:
It's probably helpful if you describe the schema here in a couple of comments. What are the keys, how do we use them, what do the records contain, etc?